### PR TITLE
Specs

### LINKED ISSUE #4: Tests
We probably want to test this.

### PR DESCRIPTION
**Pull request by jonnyarnold**
Added some tests! Closes #4.

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
_If you're new, read the [guidelines](https://github.com/reevoo/guidelines/blob/master/pull_requests.md)._

Good code is **tested**, **easy to understand** and **able to cope when things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [ ] Description explains what you have done and why.
- [ ] Tests have been written (or you have explained why you don't need tests).
- [ ] Documentation/comments have been added/updated.

**For the Reviewer**
- [ ] Tests are green on CI (or pulled/tested on your machine).
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] The code is easy to understand.

File: lib/gem-miner/github_client.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+module GemMiner
+  # A Github client with GemMiner-specific extensions.
+  class GithubClient
+
+    def initialize(github_access_token = nil)
+      @internal_client = Octokit::Client.new(
+        auto_paginate: true,
+        access_token: github_access_token
+      )
```
Comment (ionut998) [inline lib/gem-miner/github_client.rb:12]: I would raise an error if no github_access_token is passed

**Comment by ionut998** [comment]
LGTM

File: lib/gem-miner/github_client.rb (hunk 1 above)
Comment (jonnyarnold) [inline lib/gem-miner/github_client.rb:12]: If you don't, it's fine - any public project can be accessed without an access token.

**Pull request merged**